admin/src/app/Core/getToken.ts

```diff
@@ -1,5 +1,12 @@
+import { HttpHeaders } from '@angular/common/http';
+
 export const ACCESS_TOKEN_KEY: string = 'access_token';
 
 export function getToken(): string | null {
   return localStorage.getItem(ACCESS_TOKEN_KEY);
-}
\ No newline at end of file
+}
+
+export const authorizationHeader = {
+  headers: new HttpHeaders()
+  .set('Authorization', `Bearer ${getToken()}`) 
+};
```

Comment:
I'm not sure about that solution. Let me investigate it and I'll update you soon